backends-common/elasticsearch-v7/src/test/java/org/apache/james/backends/es/v7/ElasticSearchHealthCheckTest.java

```diff
@@ -68,7 +68,7 @@ private FakeClusterHealthResponse(ClusterHealthStatus clusterHealthStatus) {
             super("fake-cluster", new String[0],
```

Comment:
I don't understand this commit message: `JAMES-3492, repaijava errors in backend module tests `... Is there a typo? Can you reword it?

Reply:
Changed commit message _JAMES-3492, repair code errors after client library change in backend es module tests_